### ISSUE TITLE
"bad price"

### ISSUE THREAD

**Issue by MrLegacy119** [TheoKah/CarrotShop#4]
When setting up a iBuy or iSell sign once I punch the sign with redstone it responds in chat with a red message saying "bad price".  I've tried multiple ways of setting the price "7" "$7" "$7 each" ect, is there something i'm doing wrong?  I'm currently using the economy plugin TotalEconomy.

Also I noticed one question on the plugin was what happened to the tutorial pictures, It would really help a lot to have a step by step guide of pictures.

**Comment by TheoKah** [comment]
Did you put the price on the 4th line?

I'll see if I can make a wiki with a tutorial

**Comment by MrLegacy119** [comment]
Yes I did, I formatted it like this

[iSell]
Redstone Dust
(nothing on 3rd line)
$7

**Comment by yepidoodles** [comment]
Which Version of Sponge are you using? I'm using the latest version of SpongeForge for 5.2.0 and not having the issue you have described here.

**Comment by TheoKah** [comment]
What you wrote on the sign is correct.
Are you using vanilla signs?

If you are, I'll compile a verbose version of the plugin to see what is the problem is exaclty. If you are using vanilla signs, problem is probably on my side.

**Comment by MrLegacy119** [comment]
My current sponge version is "spongeforge-1.10.2-2254-5.2.0-BETA-2272".  I am using vanilla signs as well.

Also I've tried using a few other shop plugins recently and it seems the Quark mod has some issues with sign shops.  The quark mod adds the option to edit vanilla signs so when someone goes to buy from the shop it also brings up the edit screen for the sign, disabling this option in Quark only stops them from changing the sign but still brings up the edit screen every time they click on it.  Would you be able to add something in to bypass that?
I've tried using land claims to turn off player interactions but unfortunately that also stops them from buying for the shops.

**Comment by TheoKah** [comment]
v1.3 of CarrotShop should prevent signs edits. ( https://github.com/TheoKah/CarrotShop/releases )

Also, 1.10.2-2254-5.2.0-BETA-2272 is 23 days old, you should update with yesterday's version (2291) + latest sponge (2281)

What protection plugin do you use?

Can you try with this version:
[CarrotShop-1.3-mrlegacy119.jar.zip](https://github.com/TheoKah/CarrotShop/files/966948/CarrotShop-1.3-mrlegacy119.jar.zip)
This wont fix your problem, but this will display debug in console when you setup your sign

**Comment by MrLegacy119** [comment]
I updated to the newest sponge and forge, that seems to have fixed the bad price bug.  I also noticed that it does stop the sign editing once it's turned into a shop so that's great.  Now I just have to wait till tomorrow to see if my players can buy from it.  Also if I have interactions turned off by a protection flag would that stop people from buying from the signs?  I'm using the protection plugin called "Nations"
